Dockerfile.dev

```diff
@@ -31,24 +23,23 @@ RUN \
         libxml2 \
         git \
         cmake \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+        # Needed by the stream component \
+        ffmpeg; \
+    rm -rf /var/lib/apt/lists/*
 
-WORKDIR /usr/src
+# Disable pip version check as we pin its version in requirements.txt
+ENV PIP_DISABLE_PIP_VERSION_CHECK=1
 
 # Setup hass-release
-RUN git clone --depth 1 https://github.com/home-assistant/hass-release \
-    && pip3 install -e hass-release/
-
-WORKDIR /workspaces
+WORKDIR /usr/src/hass-release
+RUN git clone --depth 1 https://github.com/home-assistant/hass-release . \
+    && pip3 install -e .
 
 # Install Python dependencies from requirements
-COPY requirements.txt ./
+WORKDIR /usr/src/homeassistant
+COPY requirements.txt .
 COPY homeassistant/package_constraints.txt homeassistant/package_constraints.txt
 RUN pip3 install -r requirements.txt --use-deprecated=legacy-resolver
 COPY requirements_test.txt requirements_test_pre_commit.txt ./
 RUN pip3 install -r requirements_test.txt --use-deprecated=legacy-resolver
```

Comment:
Same as with the above... I don't think we need to set this stuff up here anymore.